contents/blog/mobile-session-replay.md

```diff
@@ -0,0 +1,139 @@
+---
+title: How we built mobile replay (and why it took so long)
+date: 2024-10-08
+author:
+ - ian-vanagas
+ - manoel-aranda-neto
+rootpage: /blog
+featuredImage: >- 
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-engineering-blog.png
+featuredImageType: full
+tags:
+ - Guides
+---
+
+[Session replay](/session-replay) is one of the most powerful tools for understanding user behavior. Web session replay has been a core part of PostHog for a long time now (it was built in our first hackathon), but mobile teams have had to wait longer.
+
+Finally, it's here on [iOS](/docs/session-replay/ios), [Android](/docs/session-replay/android), and [React Native](/docs/session-replay/react-native) (with [Flutter](https://github.com/PostHog/posthog-flutter/issues/69) coming soon). 
+
+What took so long? Recording them in mobile apps is much trickier than in web apps. This post goes over why and how we finally managed to make it happen.
+
+## What's so difficult about mobile session replay?
+
+Developers have complained about [the lack of good mobile replay options](https://medium.com/goodones/15-years-later-there-is-still-no-good-session-replay-for-ios-f8d335999737). Why is that the case?
```

Comment:
Rather than just linking to the source, we should summarize the common complaints – e.g. the stuff they talk about, pricing, etc. I feel like I need to read that article to understand the context.